proto/spacemesh/smesher_types.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+package spacemesh;
+
+// User provided post data
+message CommitmentUserData {
+    string path = 1;                // Path to post file(s) on a volume
+    uint64 commitment_size = 2;     // User specified commitment size in bytes
+    uint32 processor_id = 3;        // Processor to be used for smeshing. e.g. cpu, gpu0, gpu1, etc...
+    bool   append = 4;              // Append to existing file if it exists. Otherwise - overwrite
+    bool   throttle = 5;            // Throttle down init phase computations while user is interactive on system
```

Comment:
Should this be bool? Why not make it, e.g., a percentage (max. pct. of CPU to utilize), or a 1-10 scale, or number of cores?

Reply:
yes it should. It is on or off and the actual throttling happens in a lower level dynamically based on user interaction with the system per the current gpu-post init design.